async.js

```diff
@@ -13,7 +13,50 @@ const isStar = true;
  * @returns {Promise<Array>}
  */
 function runParallel(jobs, parallelNum, timeout = 1000) {
-    // асинхронная магия
+
+    return new Promise(function (resolve) {
```

Comment:
Давай уж по-взрослому:
```suggestion
    return new Promise(resolve => {
```